synapseclient/client.py

```diff
@@ -93,7 +93,7 @@
 AUTHENTICATED_USERS = 273948
 DEBUG_DEFAULT = False
 REDIRECT_LIMIT = 5
-
+NUM_THREADS = os.cpu_count() + 4
```

Comment:
Why + 4?

Reply:
additional threads for io bound tasks even if core count is low